fronts-client/src/components/card/article/ArticleBody.tsx

```diff
@@ -326,6 +333,7 @@ const articleBodyDefault = React.memo(
 									isDraggingImageOver={isDraggingImageOver}
 									isPortrait={thumbnailIsPortrait}
 									showLandscape54={showThumbnailInLandscape54}
+									showSquareThumbnail={showSquareThumbnail}
```

Comment:
This leads to the component using similar styling implemented by the feast team. I reckon the thumbnail ends up being a bit small due to the values set [here](https://github.com/guardian/facia-tool/blob/2f0f6c1180415c68748964f082fb2c882355d58a/fronts-client/src/constants/theme.ts#L157) - if others agree I'm happy to change that.